core/src/main/java/org/infinispan/configuration/global/GlobalAuthorizationConfiguration.java

```diff
@@ -31,19 +23,13 @@
  * @author Tristan Tarrant
  * @since 7.0
  */
-public class GlobalAuthorizationConfiguration implements ConfigurationInfo {
+public class GlobalAuthorizationConfiguration {
    private static final Map<String, Role> DEFAULT_ROLES;
    public static final AttributeDefinition<Boolean> ENABLED = AttributeDefinition.builder("enabled", false).immutable().build();
```

Comment:
use the `Attribute`instead of String like in other configuration classes?

Reply:
I don't understand what you mean

Reply:
something like this:
```suggestion
   public static final AttributeDefinition<Boolean> ENABLED = AttributeDefinition.builder(Attribute.ENABLED, false).immutable().build();
```